psi4/driver/procrouting/scf_proc/scf_iterator.py

```diff
@@ -489,7 +493,7 @@ def scf_iterate(self, e_conv=None, d_conv=None):
 
         # Print out the iteration
         core.print_out(
-            "   @%s%s iter %3s: %20.14f   %12.5e   %-11.5e %s\n" %
+            "   @%s%s iter %3s: %20.14f   %12.5e   %-11.5e %s \n" %
```

Comment:
why the unnecessary whitespace at the end?

Reply:
I suppose I thought it looked a bit neater this way, but I'll be happy to switch it back!

Reply:
All right, reverted!